# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/WarpBuilder.h
# Commit: f5e7bea40350
# Full Hash: f5e7bea40350979b1009753828e77b25e94b43e1
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2022-03-31 09:35:41
# Regressor Bug: 885514
# File Overlap Count: 1
# Description:
#   Bug 885514: Support finally in Warp r=jandem
#   
#   This supports any try-finally that doesn't emit a `JSOp::Retsub`. Currently that means that we don't support break, continue, or return inside the try block, but there's room to fix that in the future.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142119
# ==============================================================================

diff -r 7f75f8b705e7 -r f5e7bea40350 js/src/jit/WarpBuilder.h
--- a/js/src/jit/WarpBuilder.h	Wed Mar 30 23:19:47 2022 +0000
+++ b/js/src/jit/WarpBuilder.h	Wed Mar 30 23:19:48 2022 +0000
@@ -44,7 +44,6 @@
   _(IsGenClosing)                        \
   _(Resume)                              \
   /* try-finally */                      \
-  _(Finally)                             \
   _(Retsub)                              \
   /* Misc */                             \
   _(DelName)                             \